packages/react-native-sdk/__tests__/components/ActiveCall.test.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { mockClientWithUser } from '../mocks/client';
+import { ActiveCall } from '../../src/components';
+import { SfuModels } from '@stream-io/video-client';
+import mockParticipant from '../mocks/participant';
+import {
+  A11yButtons,
+  A11yComponents,
+  A11yIcons,
+} from '../../src/constants/A11yLabels';
+import { ViewToken } from 'react-native';
+import { mockCall } from '../mocks/call';
+import { act, render, screen, within } from '../utils/RNTLTools';
+
+console.warn = jest.fn();
+jest.useFakeTimers();
+
+enum P_IDS {
+  LOCAL_1 = 'local-1',
+  REMOTE_1 = 'remote-1',
+  REMOTE_2 = 'remote-2',
+}
+const simulateOnViewableItemsChanged = async (
+  viewableItems: Array<ViewToken>,
+) => {
+  const flatList = await screen.findByLabelText(
+    A11yComponents.CALL_PARTICIPANTS_LIST,
+  );
+  await act(() => {
+    flatList.props.onViewableItemsChanged({
+      viewableItems,
+    });
+    // Advance pending timers to allow the FlatList to rerender
+    // This is needed because of useDebouncedValue we use in
+    // forceUpdateValue to force rerender the FlatList
+    jest.advanceTimersByTime(500);
```

Comment:
This is why I added this:
Advance pending timers to allow the FlatList to rerender
This is needed because of useDebouncedValue we use in
forceUpdateValue to force rerender the FlatList